### ISSUE TITLE
Need Ghost 2.0 example

### ISSUE THREAD

**Issue by TooTallNate** [zeit/now-examples#50]

**Comment by LDubya** [comment]
a little confused because @grikomsn had a pull request in September yet it can't be found anywhere

**Comment by antony** [comment]
Would be great to see the v2 example! Having a nightmare of a time trying to do it myself.

**Comment by antony** [comment]
@grikomsn I'm actually looking for a base where I can add the GCP Storage adaptor for content, and an external pgSql database for the db - meaning that hopefully maintenance will be quite low!

Thanks for your efforts :)

**Comment by mittalyashu** [comment]
Any updates on this issue?

**Comment by antony** [comment]
@mittalyashu I started doing a port of Ghost to Now v2.0 but it refuses to deploy and doesn't output any errors that I can use to determine what the error is, and I'm not sure how to progress from there, so I didn't continue.

I'm happy to share the project so that somebody else can collaborate if there is interest.

**Comment by mittalyashu** [comment]
@antony You can share the link to the repository.

**Comment by LDubya** [comment]
@antony there is interest

**Comment by antony** [comment]
@mittalyashu @LDubya here's what I have so far:

https://github.com/beyonk-adventures/now-v2-ghost

Here's the spectrum thread I started regarding the issue, it contains stack traces etc: https://spectrum.chat/?t=59bcb54f-6a0a-465b-b630-a2f9b9d11c1e

**Comment by LDubya** [comment]
@antony thanks! does this repo work on Now V1 as-is?

**Comment by antony** [comment]
@LDubya no - the now v1 example can be seen in the PR closed by @timothyis above.

Important note - the examples both use google cloud storage for storing files (as now has an ephemeral filesystem), so you need to add some google credentials into the project where indicated if you want it to work once deployed.

**Comment by etamity** [comment]
I managed to override the default config content path.

Here is the way:
```js
const { join } = require('path');
const defualtConfig = require('./node_modules/ghost/core/server/config');

const contentPath = join(__dirname, 'content');
defualtConfig.set('paths:contentPath', contentPath);
defualtConfig.set('paths:internalStoragePath', join(contentPath, 'adapters/'));
defualtConfig.set(
  'database:connection:filename',
  join(contentPath, 'data', 'ghost-dev.db')
);

const ghost = require('ghost');
const express = require('express');
const { join } = require('path');

async function bootstrap() {
  const server = express();
  const ghostServer = await ghost();
  server.use('/', ghostServer.rootApp);
  ghostServer.start(server);
  return server;
}

```

**Comment by meshfields** [comment]
This aint working anymore.

```
⇒  now
> Deploying ~/dev/now-v2-ghost under xyz
> Using project now-v2-ghost
> Error! Your `package.json` file is missing a `build` property inside the `script` property.
More details: https://zeit.co/docs/v2/advanced/platform/frequently-asked-questions#missing-build-script
```
I added the recquired build section but then Ghost Starts Up inside the Build Step of Now. We need Now to deploy Ghost not stop after the building step. Any ideas? Who can work with me on this?

**Comment by elitekode2** [comment]
what happens when you add a build property?

**Comment by meshfields** [comment]
I add **build property** in package.json: which is `node index'`and then we hang in **Zeit logs** after Ghost started.

```
019-09-17T23:17:35.519Z  [2019-09-17 23:17:35] INFO Ghost is running in development...
2019-09-17T23:17:35.520Z  [2019-09-17 23:17:35] INFO Listening on: 127.0.0.1:2368
2019-09-17T23:17:35.521Z  [2019-09-17 23:17:35] INFO Url configured as: http://localhost:2368/
2019-09-17T23:17:35.522Z  [2019-09-17 23:17:35] INFO Ctrl+C to shut down
2019-09-17T23:17:35.523Z  [2019-09-17 23:17:35] INFO Ghost boot 4.935s
```

Good news: I escalated this with official Zeit support and they are on it. They want to bring Zero Config to Ghost and Jekyll if possible.

**Comment by togume** [comment]
Hi all - Ghost 3.0 has been released. Any updates on being able to host it with Now + Zero Config?

**Comment by rauchg** [comment]
We probably won't be a **GREAT** hosting platform for Ghost the engine itself (although with functions I suppose you technically _could_ do it.

What I would do instead is use ZEIT Now to host the frontend, for example using static site generation that queries a cloud-hosted version of Ghost.

**Comment by LDubya** [comment]
What are the advantages of hosting a statically generated Ghost with Zeit over, say GitHub Pages?

**Comment by rauchg** [comment]
High level: ZEIT Now is an all-in-one solution and workflow for teams, that focuses **primarily** on deployment and hosting. GH Pages is a nice feature / convenience and not a core competency.[1]

Technically, ZEIT Now…
- Provides built-in CI/CD. This means that when you `git push`, we can build and then deploy to the edge.
- Features a built-in CDN network **optimized** for fast-changing static assets. For example, when a new deployment or build is made, or when a page changes, that change is reflected instantly, worldwide.
- Unlike GitHub Actions, our built-in deployment solution requires **no configuration**. Take your favorite framework, like Next.js and Gatsby, and just run `npx now`

[1] I'll grant that it's a well executed and popular feature!

**Comment by rauchg** [comment]
We experimented with that (serverless docker), but ended up with the conclusion that hosting something like Ghost which would be susceptible to a cold start to render some HTML is probably not the best idea…

I'd much rather interoperate with something like https://ghost.org/pricing/ where for $30/mo you'll get updates and guaranteed **hot** performance. And then on our side we will do the best to serve pages generated from it. Global, multi-framework, zero-config and with the best DX and lowest latency.

In short, you'll get a world-class experience and outcome. With GCR you are more likely to get an awkward in-between (more maintenance work on your side, slower perf, slower builds, not global)
